_layouts/author.html

```diff
@@ -12,7 +12,12 @@
 <div class="ui vertical stripe segment authors-page">
   <div class="ui middle aligned stackable text container">
 
-  <h1>{{ page.author }}</h1>
+  <h1 class="author-header">
+    {{ page.author }}
+    {% if site.RIMdev[page.author].twitter %}
```

Comment:
Should this be looking for `previous: true` instead?

Reply:
oh yeah, copy and paste error